devices/microsoft/research/jmspeechsynthesisv33.json

```diff
@@ -0,0 +1,15 @@
+{
+  "id": "microsoft-research-jmspeechsynthesisv331vl",
+  "name": "JM Speech Synthesis v3.3",
+  "company": "Microsoft Research",
+  "description": "JM Speech Synthesis v3.3",
+  "repo": "https://github.com/microsoft/jacdac-msr-modules",
+  "services": [
+    302307733
+  ],
+  "productIdentifiers": [
+    934541191
+  ],
+  "version": "l",
+  "designIdentifier": "1"
```

Comment:
Can you use server? 1.0

Reply:
I didn't enter these fields, as described above

Reply:
Correct revision is 3.3, no design identifier has been assigned to this proto board.

Reply:
we could probably use design id 61 as it is a previous version of the latest mikrobus module